src/main/java/roomescape/controller/ConsoleController.java

```diff
@@ -0,0 +1,106 @@
+package roomescape.controller;
+
+import java.time.Clock;
+import java.util.function.Supplier;
+import roomescape.dto.CreateReservationDto;
+import roomescape.dto.CreateReservationTimeDto;
+import roomescape.repository.MemoryReservationRepository;
+import roomescape.repository.MemoryReservationTimeRepository;
+import roomescape.service.ReservationService;
+import roomescape.service.ReservationTimeService;
+import roomescape.view.Command;
+import roomescape.view.InputView;
+import roomescape.view.OutputView;
+
+public class ConsoleController {
+
+    private final InputView inputView;
+    private final OutputView outputView;
+    private final ReservationService reservationService;
+    private final ReservationTimeService reservationTimeService;
+
+    public ConsoleController() {
+        inputView = new InputView();
+        outputView = new OutputView();
+        MemoryReservationTimeRepository memoryReservationTimeRepository = new MemoryReservationTimeRepository();
+        MemoryReservationRepository memoryReservationRepository = new MemoryReservationRepository();
```

Comment:
MemoryReservationTimeRepository, MemoryReservationRepository라는 구체적인 클래스를 의존하는 것 보다 ReservationRepository, ReservationTimeRepository와 같은 인터페이스를 의존하면 더욱 유연하게 활용 가능할 것 같아요.

```suggestion
        ReservationTimeRepository memoryReservationTimeRepository = new MemoryReservationTimeRepository();
        ReservationRepository memoryReservationRepository = new MemoryReservationRepository();
```

Reply:
넵 반영했습니다!